common/sass/styles/scaffolding.scss

```diff
@@ -34,6 +34,20 @@ body {
   height: 100%;
 }
 
+#app {
+  height: 100%;
+}
+
+.page-layout {
+  height: 100%;
```

Comment:
Shouldn't this and #app be `min-height`? Will anything weird happening with overflow if it exceeds the page height?

Reply:
Nope, nothing weird would happen. Typically, all components get added to `<body />` and when it overflows, a scrollbar appears on `<body />`. Since it's a react app, we also wrap all our components in `<div id="app" />`. Then everything is wrapped once more in `<div className="page-layout" />`. Adding the height 100% to these 3 DOM elements ensures the content will be the height of the viewport, and any overflow will show a scrollbar. 

Really, the `height: 100%;` on `.page-layout` and `#app` is the same as inheriting the height from `<body />` with `height: inherit;`